src/db/service/user.service.ts

```diff
@@ -147,14 +147,18 @@ export default class UserService {
 		}
 	}
 
-	async validatePassword({ email, password }: { email: string; password: string }) {
-		const user = await this.userDocumentCollection.findOne({ email });
-
+	async validatePassword(body: any) {
```

Comment:
There is no test for getting/searching by username yet. Can you add at least one so that we know it is working?